docs/book/src/topics/managedcluster.md

```diff
@@ -201,6 +203,41 @@ should be fairly clear from context.
 | networkPlugin             | azure, kubenet                |
 | networkPolicy             | azure, calico                 |
 
+### AKS Addons
+
+AKS addons are enabled via the [addonProfiles](https://learn.microsoft.com/cli/azure/aks/addon?view=azure-cli-latest#az-aks-addon-list-available) specification. For more information about required config values and addons which may not be listed below, look for the `*ADDON_NAME` values in [this code](https://github.com/Azure/azure-cli/blob/main/src/azure-cli/azure/cli/command_modules/acs/_consts.py).
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureManagedControlPlane
+metadata:
+  name: my-cluster-control-plane
+spec:
+  location: southcentralus
+  resourceGroupName: foo-bar
+  subscriptionID: 00000000-0000-0000-0000-000000000000 # fake uuid
+  version: v1.21.2
+  addonProfiles:
+  - name: "omsagent"
+    enabled: true
+    config:
+     "logAnalyticsWorkspaceResourceID": /subscriptions/<subscription-id>/resourcegroups/<resource-group>/providers/microsoft.operationalinsights/workspaces/<workspace-id>
+  - name: azurepolicy
+    enabled: true
+```
+
+| addon name                | YAML value                | Config required? |
```

Comment:
are we confident that these values are pretty static? if not I wonder if we should even duplicate the info in here instead of just linking to a source of truth or AKS docs

Reply:
I'm pretty confident these values are pretty static.  If I could find a different source of truth, then I'd point there.  So... this becomes the source of truth instead.